ppl/cmd/zqd/listen/command.go

```diff
@@ -151,6 +152,9 @@ func (c *Command) Run(args []string) error {
 }
 
 func (c *Command) init() error {
+	if err := fromenv.Unmarshal(&c.conf); err != nil {
```

Comment:
Wouldn't this overwrite conf variables from the command line. IIRC the standard command line arg > env > config file. No?

Reply:
I suppose this ok for now, since the auth parameters can be configured via cli arg. Something to think about going forward though.